ssh/DOCS.md

```diff
@@ -65,14 +72,22 @@ Specifies whether TCP forwarding is permitted or not.
 
 **Note**: _Enabling this option lowers the security of your SSH server! Nevertheless, this warning is debatable._
 
+
 ## Network
-To enable ssh access via the network, you need to enter the port number ‘22’ or the port you want to use. This will map that port from the hassio host into the running “Terminal & SSH” container.
+
+To enable ssh access via the network, you need to specify which external port to use. 
+The number you enter, will be used to map that port from the hassio host into the 
+running "Terminal & SSH" container.
+
+The standard port for the SSH protocol is `22`.
+
```

Comment:
Not needed extra new line.

Reply:
I need help to disambiguate this comment, as there are four new line characters in play in this comment, and I can see that at least three could be seen as "extra new line". I'm unsure which one you refer to, and guess it's one of these:
* the one separating the two sections (line 81), or
* the one adding white space between the next 2nd level headline (line 83).

Reply:
multiple new lines have no effect. please remove them as requested. Thanks.

Reply:
We disagree, I've updated as per your request.